src/gas/controllers/Clubs/GetController.ts

```diff
@@ -0,0 +1,11 @@
+// import Constants from "../../shared/Constants";
+
+import GetModel from "../../models/Clubs/GetModel";
+
+export default class GetController {
+  private get = new GetModel();
```

Comment:
```suggestion
  private model = new GetModel();
```
これでも良い気がしてる
verb.verb って個人的には違和感

Reply:
DONE